File: testprojects/src/java/org/pantsbuild/testproject/nocache/Hello.java
// Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
// Licensed under the Apache License, Version 2.0 (see LICENSE).

package org.pantsbuild.testproject.nocache;


public class Hello {

  public static void main(String[] args) {
    System.out.println("For some reason, we don't want to cache this in the build cache");
  }
}

File: examples/src/python/example/python_distribution/hello/fasthello/hello_package/hello.py
# coding=utf-8
# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

from __future__ import absolute_import, division, print_function, unicode_literals

import c_greet
import cpp_greet


def hello():
  return '\n'.join([
    c_greet.c_greet(),
    cpp_greet.cpp_greet(),
  ])
